services/util/MessageCommunicationCobs.hpp

```diff
@@ -14,9 +14,9 @@ namespace services
     {
     public:
         template<std::size_t MaxMessageSize>
-            using WithMaxMessageSize = infra::WithStorage<infra::WithStorage<MessageCommunicationCobs,
```

Comment:
Hmm I really don't like this reformatting.

Reply:
This is improved when we set ColumnLimit.